doc/modules/classes.rst

```diff
@@ -881,7 +881,7 @@ details.
 
    metrics.adjusted_mutual_info_score
    metrics.adjusted_rand_score
-   metrics.calinski_harabaz_score
+   metrics.calinski_harabasz_score
```

Comment:
at the bottom of this page there is a list of deprecated functions etc. Please add the misspelled version there.